magento/magento2
# Fix static content deploy command error on php8.1

### Description (*)

The pathinfo() function does not accept null as a valid $path parameter on php 8.1, causing an exception when executing `bin/magento setup:static-content:deploy --area frontend --theme Them/name --language pt_BR -f`. This was fixed by adding a check on the `Magento\Framework\View\Asset\PreProcessor\FileNameResolver::resolve` function.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#36020

### Manual testing scenarios (*)

1. Make sure your environments is running PHP 8.1
2. Create a custom theme and Luma as a parent
3. Run below command in terminal
bin/magento setup:static-content:deploy --area frontend --theme Customtheme/mytheme --language pt_BR -f

### Questions or comments


### Contribution checklist (*)
